components/radio/style/index.less

```
@@ -174,7 +174,7 @@ span.@{radio-prefix-cls} + * {
   display: inline-block;
   height: @btn-height-base;
   margin: 0;
-  padding: 0 @padding-md - 1px;
+  padding: @radio-button-wrapper-padding-vertical @radio-button-wrapper-padding-horizontal;
```

```suggestion
  padding: @radio-button-padding-vertical @radio-button-padding-horizontal;
```

这样好了，这里从语义上用户不需要感知 wrapper。

---

已修改